Makefile

```diff
@@ -8,6 +8,8 @@ FIRST_GOPATH := $(firstword $(subst :, ,$(shell go env GOPATH)))
 OS ?= $(shell uname -s | tr '[A-Z]' '[a-z]')
 ARCH ?= $(shell uname -m)
 GOARCH ?= $(shell go env GOARCH)
+GOFLAGS := -tags=fips_enabled
```

Comment:
Do we need this build tag here?

Reply:
Looked a little closer, and I don't think it's necessary